src/analysis/retail/warlock/demonology/modules/guide/ResourceUsage.tsx

```diff
@@ -5,12 +5,12 @@ function ResourceUsage({ modules }: GuideProps<typeof CombatLogParser>) {
   return (
     <Section title="Resource Use">
       <SubSection title="Soul Shards">
-        <p>
+        <>
```

Comment:
You don't technically need the `<>` here as the children are already wrapped by the parent `<Subsection>` component.